tools/fast/series/wait_for_deal_state.go

```diff
@@ -9,12 +9,12 @@ import (
 
 // WaitForDealState will query the storage deal until its state matches the
 // passed in `state`, or the context is canceled.
-func WaitForDealState(ctx context.Context, client *fast.Filecoin, deal *storagedeal.Response, state storagedeal.State) error {
+func WaitForDealState(ctx context.Context, client *fast.Filecoin, deal *storagedeal.Response, state storagedeal.State) (dr *storagedeal.Response, err error) {
```

Comment:
Non-Blocking: We're moving away from named return values in general. I think it's adding an extra layer of indirection here. My preference would be to not name the parameter and to return `dr` directly on line 21.